example/benchmark/Benchmark.toml

```diff
@@ -0,0 +1,34 @@
+version = "0"
+
+[[stages]]
+name = "benchmark"
+deps = ["scripts"]
+
+    [stages.run]
+        metrics = ["time"]
+        out = ["results.csv"]
+        driver = "csv"
```

Comment:
Currently, it outputs

```text
id,time
dilate_juliaimages_morphology,0.623235
erode_juliaimages_morphology,0.627855
dilate_skimage_morphology,1.5773831250000003
```

and if I change to `metrics = ["time", "memory"]`, then it becomes

```text
id,time,memory
dilate_juliaimages_morphology,0.636088,1280096
erode_juliaimages_morphology,0.641488,1280096
dilate_morphology_skimage,1.8287903700000008,

```


cc: @ashwani-rathee

Reply:
I should make `metrics` an optional field and defaults to all results. Or just remove this field.